great_expectations/data_context/types/data_context_variables.py

```diff
@@ -0,0 +1,111 @@
+import enum
+from abc import abstractmethod
+from dataclasses import dataclass
+from typing import Any, Optional
+
+import requests
+
+
+class VariableSchema(enum.Enum):
```

Comment:
This is used to ensure no magic strings but it is a bit of code duplication I don't love.